### PR TITLE
Fix/hck 4445 constraints

### PR DESCRIPTION
**Pull request by ArzamastsevVladyslav**

### CODE REVIEW

File: properties_pane/entity_level/entityLevelConfig.json [hunk 1]
```diff
@@ -872,20 +872,26 @@ making sure that you maintain a proper JSON format.
 						"propertyKeyword": "constrDescription",
 						"propertyTooltip": "description",
 						"propertyType": "details",
-						"template": "textarea"
+						"template": "codeEditor",
```
Comment (Vitalii4as) [inline properties_pane/entity_level/entityLevelConfig.json:875]:
```
"template": "textarea",
"markdown": true
```

File: properties_pane/entity_level/entityLevelConfig.json [hunk 2]
```diff
@@ -894,7 +900,10 @@ making sure that you maintain a proper JSON format.
 						"propertyTooltip": "comments",
 						"addTimestampButton": false,
 						"propertyType": "details",
-						"template": "textarea"
+						"template": "codeEditor",
```
Comment (Vitalii4as) [inline properties_pane/entity_level/entityLevelConfig.json:903]: "template": "textarea",
"markdown": true

**Review by Vitalii4as — CHANGES_REQUESTED** [review]

File: properties_pane/entity_level/entityLevelConfig.json (hunk 1 above)
Comment (ArzamastsevVladyslav) [reply properties_pane/entity_level/entityLevelConfig.json:875]: Code editor has "HTML" preview as well, so the issue is resolved

File: properties_pane/entity_level/entityLevelConfig.json (hunk 2 above)
Comment (ArzamastsevVladyslav) [reply properties_pane/entity_level/entityLevelConfig.json:903]: Code editor has "HTML" preview as well, so the issue is resolved

**Review by Vitalii4as — APPROVED** [review]

**Pull request merged**